gramex/handlers/mlhandler.py

```diff
@@ -40,6 +44,8 @@
     'nums': [],
     'cats': [],
     'target_col': None,
+    'CV': True,
+    'CVargs': []
```

Comment:
Let's have a single argument, `cv`, which can take any value, i.e in gramex.yaml, users should be able to write any of the following.

```yaml
cv: false   # disable cross validation
cv: 5        # Use 5 folds
cv:
  cv: 8   # Use 8 folds
  n_jobs: -1  # with an optional other parameter.
```